exec/csnex/Cell.cs

```diff
@@ -309,6 +313,59 @@ public Boolean Equals(Cell x)
             }
             return false;
         }
+
+        public static void CopyCell(Cell dest, Cell source)
+        {
+            Debug.Assert(source != null);
+            Debug.Assert(dest != null);
```

Comment:
Should you also assert that the source type is the same as the destination type? What happens if it isn't?

Reply:
No.  Since most new cells start off life as a `Type.None`, I want them to be able to morph here.  If they are different types, then it will literally copy the source over the destination cell blindly, which is ok since it also types the type member.

Reply:
I must be missing it, where is the type member copied?